.buildkite/commands/configure-git-for-release-management.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eu
+
+# Git command line client is not configured in Buildkite. Temporarily, we configure it in each step.
+# Later on, we should be able to configure the agent instead.
+curl -L https://api.github.com/meta | jq -r '.ssh_keys | .[]' | sed -e 's/^/github.com /' >> ~/.ssh/known_hosts
+git config --global user.email "mobile+wpmobilebot@automattic.com"
+git config --global user.name "Buildkite"
```

Comment:
Nitpick, but Buildkite seems a bit generic as the user name in the context of release automation 🤔 

What do you think of 

```suggestion
git config --global user.name "Automattic Release Bot"
```